packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -526,29 +568,7 @@ const CFDPasswordForm = ({
                     <div className='cfd-password-modal__content dc-modal__container_cfd-password-modal__body'>
                         {!should_set_trading_password && (
                             <Text size='xs' className='dc-modal__container_cfd-password-modal__account-title'>
-                                {account_type.category === 'real' && (
-                                    <Localize
-                                        i18n_default_text='Enter your {{platform}} password to add a {{platform_name}} {{account}} {{jurisdiction_shortcode}} account.'
-                                        values={{
-                                            platform: getCFDPlatformLabel(platform),
-                                            platform_name: getCFDPlatformNames(platform),
-                                            account: !show_eu_related_content
-                                                ? accountTitle(account_type.category)
-                                                : '',
-                                            jurisdiction_shortcode: showJuristiction(),
-                                        }}
-                                    />
-                                )}
-                                {account_type.category === 'demo' && (
-                                    <Localize
-                                        i18n_default_text='Enter your {{platform}} password to add a {{platform_name}} {{account}} account.'
-                                        values={{
-                                            platform: getCFDPlatformLabel(platform),
-                                            platform_name: getCFDPlatformNames(platform),
-                                            account: accountTitle(account_type.category),
-                                        }}
-                                    />
-                                )}
+                                {getPasswordModalTitle()}
```

Comment:
Turn this to a `Component`. Invoking this as a function will make the babel-jsx-transform will treat it as a hook.

Refer to this: https://deriv-group.slack.com/archives/C05NSB1LPMX/p1698220418571269

Reply:
@yashim-deriv this is done for this particular usage. there are other similar cases in the file, which i am ignoring for now. hope this is fine. 🙏🏼